### PR TITLE
[src/doc/manifest] Add section on Migrating to `gitignore`-like pattern matching

### PR DESCRIPTION
**Pull request by behnam**
Tracking issue: https://github.com/rust-lang/cargo/issues/4268

### CODE REVIEW

**Comment by behnam** [comment]
I have not seen any similar notices on <http://doc.crates.io/> which made me wonder if this is the right place to notify the change, which will eventually update this part of the document.

What do you think, @alexcrichton, others?

File: src/doc/manifest.md [hunk 1]
```diff
@@ -81,6 +81,20 @@ necessary source files may not be included.
 
 [globs]: http://doc.rust-lang.org/glob/glob/struct.Pattern.html
 
+### Migrating to `gitignore`-like pattern matching
+
+The current interpretation of these configs is based on UNIX Globs, as
+implemented in the [`glob` crate](https://crates.io/crates/glob).  Cargo We want
```
Comment (alexcrichton) [inline src/doc/manifest.md]: Stray "Cargo" here on this line?

**Comment by alexcrichton** [comment]
Looks great!

File: src/doc/manifest.md (hunk 1 above)
Comment (behnam) [inline src/doc/manifest.md]: Oops! Fixed!

**Pull request merged**